website/docs/guides/markdown-features/markdown-features-code-blocks.mdx

```diff
@@ -78,7 +78,7 @@ Because a Prism theme is just a JS object, you can also write your own theme if
 
 ### Supported Languages {#supported-languages}
 
-By default, Docusaurus comes with a subset of [commonly used languages](https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/vendor/prism/includeLangs.js).
+By default, Docusaurus comes with a subset of [commonly used languages](https://github.com/FormidableLabs/prism-react-renderer/blob/master/packages/generate-prism-languages/index.ts).
```

Comment:
I'm not sure it's the proper replacement link (nor if it actually still exists). Waiting for an answer in https://github.com/FormidableLabs/prism-react-renderer/issues/190

Old list: https://github.com/FormidableLabs/prism-react-renderer/blob/v1.3.0/src/vendor/prism/includeLangs.js